my-sms/db/schema.rb

```diff
@@ -23,6 +22,13 @@
     t.string   "gender"
     t.datetime "created_at",  :null => false
     t.datetime "updated_at",  :null => false
+    t.integer  "title_id"
+  end
+
+  create_table "titles", :force => true do |t|
+    t.string   "name"
+    t.datetime "created_at", :null => false
+    t.datetime "updated_at", :null => false
```

Comment:
Can rails 3.2 use `timestamps` for these fields?

Reply:
Ignore this, I'm looking at the wrong file

Reply:
Seems like it, with some difficulty.
As far as I can tell Rail prefers to use datetime if available on the database. I defined t.timestamps in the migration file and haven't touched the schema.rb file.